test/tokenizer.test.ts

```diff
@@ -97,7 +97,7 @@ Deno.test("Tokenizer (comment)", () => {
   assertEquals(tokens, [
     ["string", "<h1> ", 0],
     ["comment", " {{ message }} ", 5],
-    ["string", " </h1>", 23],
+    ["string", " </h1>", 26],
```

Comment:
Not sure if this was intentional, but 

```
<h1> {{# {{ message }} #}} </h1>
                       ^  `- This is the 26th character
                       `-- This is the 23rd character
```

I reckon 26 is right?

Reply:
Yes, it should be 26. Perhaps it was a bug in the previous tokenizer.